src/style.css

```diff
@@ -1,3 +1,75 @@
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.header {
+  width: 100%;
+  margin-left: 0;
+  margin-right: 0;
+  box-sizing: border-box;
+}
+
+.header__logo {
+  display: flex;
+  align-items: center;
+  min-width: 0;
+}
+
+.header__container {
+  display: flex;
+  align-items: center;
+  box-shadow: 0 2px 8px rgba(0, 0, 0, 0.08);
+  padding: 0 50px;
+}
+
+.nav {
+  margin-left: auto;
+}
+
+.nav__list {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+  display: flex;
+  flex-direction: row;
+  margin-left: auto;
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  position: relative;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #000;
+  height: 60px;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.nav__link.is-active::after {
+  position: absolute;
+  content: '';
+  border-radius: 8px;
+  width: 37px;
+  height: 4px;
+  bottom: -1px;
+  right: 0;
+  background: #00acdc;
```

Comment:
Create a variable for this color and use it everywhere 
<img width="373" alt="image" src="https://github.com/user-attachments/assets/a6d1b8e3-e18f-4276-949d-73f80e9a39cc" />